docker/volumeplugin/plugin.go

```diff
@@ -64,12 +66,14 @@ type RancherStorageDriver struct {
 	Basedir         string
 	Scope           string
 	CreateSupported bool
+	InBandAttach    bool
 	Command         string
 	client          *client.RancherClient
 	state           *RancherState
 	mounter         *mount.SafeFormatAndMount
 	FsType          string
 	cli             *dockerClient.Client
+	devices         map[string]string
```

Comment:
This concerns me.  Doesn't this effectively make the driver stateful now?  What happens on restart?